.ci/bump-version.sh

```diff
@@ -1,9 +1,68 @@
 #!/bin/bash
 set -e
 
-VERSION=$1
-echo VERSION="$VERSION"
+if [[ -z $1 ]]; then
+  echo "version is not set"
+  echo "Usage: $BASH_SOURCE <version>"
+  exit 1
+fi
+
+NEW_VERSION=$1
+echo NEW_VERSION="$NEW_VERSION"
+
+if ! [[ "$NEW_VERSION" =~ ^[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
Why `[0-9]` and not `\d`?

Reply:
`\d` does not work.

This answer explains why: https://stackoverflow.com/a/17968116/9553927
![Screenshot from 2023-02-05 18-20-07](https://user-images.githubusercontent.com/23459549/216834181-301a1958-5e8d-4ead-a2dd-b02132a34733.png)